### ISSUE TITLE
Handle server ws ping

### ISSUE THREAD

**Issue by rabzu** [OutOfAxis/pixelart-player-bs#20]
I enabled new [feature](https://doc.akka.io/docs/akka-http/current/server-side/websocket-support.html#automatic-keep-alive-ping-support) in our server that sends Ping messages to ws client:

Will this be handled automatically or we will need to disable ping messages from the client and implement ping message handling from the server? @ilyai

**Comment by ilyai** [comment]
Yes, according to documentation Pong messages are automatically sent in response to ping messages: https://github.com/websockets/ws#how-to-detect-and-close-broken-connections
